src/App.css

```diff
@@ -19,20 +19,116 @@
 }
 
 .App-header {
-  background-color: #282c34;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
   color: white;
+  margin: auto;
 }
 
 .App-link {
   color: #61dafb;
 }
 
+.Background-logo {
+  background-color: darkred;
+  background-image: url('./Background.jpg');
+  background-attachment: fixed;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: flex;
+}
+
+.Main-text {
+    text-shadow: 2px 0 0 #fff, -2px 0 0 #fff, 0 2px 0 #fff, 0 -2px 0 #fff, 1px 1px #fff, -1px -1px 0 #fff, 1px -1px 0 #fff, -1px 1px 0 #fff;
```

Comment:
There is an easier way to describe this shadow, I guess

Reply:
Duplicated conversation.